app/data_models/questionnaire_store.py

```diff
@@ -10,18 +12,21 @@
 class QuestionnaireStore:
     LATEST_VERSION = 1
 
-    def __init__(self, storage, version=None):
+    def __init__(self, storage: Any, version: int = None):
+        # Storage type hint needs to be Any due to circular import
```

Comment:
Circular import when typing can be avoided by using the type hinting flag.
```py
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from something import stuff
```

Reply:
I was able get pass the circular import problem by just importing `EncryptedQuestionnaireStorage` type, possibly some other typing fixes helped. Thanks.